fe/fe-core/src/main/java/org/apache/doris/analysis/HashDistributionDesc.java

```diff
@@ -80,6 +80,9 @@ public void analyze(Set<String> colSet, List<ColumnDef> columnDefs) throws Analy
                     if (columnDef.getType().isScalarType(PrimitiveType.STRING)) {
                         throw new AnalysisException("String Type should not be used in distribution column["
                                 + columnDef.getName() + "].");
+                    } else if (columnDef.getType().isArrayType()) {
```

Comment:
Maybe we should move all these checks to `toDistributionInfo()` ?

Reply:
done